stays/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from customers.models import Person, Provider, Pet
+
+
+class Stay(models.Model):
+    owner = models.ForeignKey(Person, on_delete=models.CASCADE)
+    provider = models.ForeignKey(Provider, on_delete=models.CASCADE)
+    start_date = models.DateField()
+    end_date = models.DateField()
+    pets = models.ManyToManyField(Pet)
+
+    def __str__(self):
+        return f"Stay for - {self.owner} - {self.id}"
+
+    def __repr__(self):
+        return f"Stay for - {self.owner} - {self.id}"
```

Comment:
The goal of `__str__` is to be readable, the goal of `__repr__` is to be unambiguous. I would give more informations about the model on the `__repr__` method and diferentiate it from the `__str__`. You could define something like `"Stay(id=self.id, owner={self.owner}, provider={self.provide}, ...)"` for the `__repr__`